docs: update cookbooks #138 (#3098)
jina-ai/jina: .github/2.0/cookbooks/Math.md

# Jina Math

> Jina offers several Math helpers used by it's core logic and that users can use for their own purposes (for example, in their executors).

Reviewer: ```suggestion
Jina offers several Math helpers used by its core logic that users can use for their own purposes (for example, in their Executors).
```
